mozci/data/sources/activedata/queries/group_durations.query

```diff
@@ -5,9 +5,12 @@ groupby:
 select:
     - {name: result_duration, value: result.duration, aggregate: sum}
 limit: 100000
+destination: url
 where:
     and:
+        - ne: {treeherder.tier: 3}
         - ne: {result.status: "SKIP"}
-        - in: {task.id: {$eval: task_id}}
+        - in: {repo.branch.name: {$eval: branch}}
```

Comment:
Nit: since it's a single branch, you can just do `- eq: {repo.branch.name: {$eval: branch}}`.